Tue Jun 13 08:42:56 CEST 2023 Thread[JavaFX Application Thread,5,main] Cleanup action starting
java.sql.SQLException: Database 'db' not found.
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:225)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:220)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(EmbedConnection.java:3215)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(EmbedConnection.java:772)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:442)
	at org.apache.derby.engine/org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.engine/org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at org.apache.derby.engine/org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.engine/org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.engine/org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.engine/org.apache.derby.iapi.jdbc.AutoloadedDriver.connect(AutoloadedDriver.java:136)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)
	at com.example.stayfit/com.example.stayfit.model.Database.openConnection(Database.java:13)
	at com.example.stayfit/com.example.stayfit.stayfitApp.start(stayfitApp.java:33)
	at javafx.graphics@18.0.2/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:847)
	at javafx.graphics@18.0.2/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:484)
	at javafx.graphics@18.0.2/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at javafx.graphics@18.0.2/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at javafx.graphics@18.0.2/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
	at javafx.graphics@18.0.2/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at javafx.graphics@18.0.2/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:184)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: ERROR XJ004: Database 'db' not found.
	at org.apache.derby.commons/org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 26 more
============= begin nested exception, level (1) ===========
ERROR XJ004: Database 'db' not found.
	at org.apache.derby.commons/org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:225)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:220)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(EmbedConnection.java:3215)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(EmbedConnection.java:772)
	at org.apache.derby.engine/org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:442)
	at org.apache.derby.engine/org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.engine/org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at org.apache.derby.engine/org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.engine/org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.engine/org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.engine/org.apache.derby.iapi.jdbc.AutoloadedDriver.connect(AutoloadedDriver.java:136)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)
	at com.example.stayfit/com.example.stayfit.model.Database.openConnection(Database.java:13)
	at com.example.stayfit/com.example.stayfit.stayfitApp.start(stayfitApp.java:33)
	at javafx.graphics@18.0.2/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:847)
	at javafx.graphics@18.0.2/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:484)
	at javafx.graphics@18.0.2/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at javafx.graphics@18.0.2/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at javafx.graphics@18.0.2/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
	at javafx.graphics@18.0.2/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at javafx.graphics@18.0.2/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:184)
	at java.base/java.lang.Thread.run(Thread.java:833)
============= end nested exception, level (1) ===========
Cleanup action completed
----------------------------------------------------------------
Tue Jun 13 08:42:56 CEST 2023: Shutting down Derby engine
----------------------------------------------------------------
